specification/app/Microsoft.App/models/code-execution-file.tsp

```diff
@@ -5,33 +5,33 @@ using TypeSpec.Rest;
 namespace Microsoft.App;
 
 @doc("Code execution file resource collection.")
-model CodeExecutionFileCollection {
+model SessionResourceFileCollection {
   @doc("The files uploaded.")
-  value: CodeExecutionFile[];
+  value: SessionResourceFile[];
 }
 
 @doc("Code execution file resource.")
 @parentResource(SessionPool)
 @resource("files")
-model CodeExecutionFile {
```

Comment:
Currently we can upload/download/get file with relative file path also. how that is being handled under `fileName` parameter ?